dlp/README.rst

```diff
@@ -59,6 +59,26 @@ Windows
     <your-env>\Scripts\activate
     <your-env>\Scripts\pip.exe install gapic-google-cloud-dlp-v2beta1
 
+Preview
+~~~~~~~
+
+DlpServiceClient
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: py
```

Comment:
What is the deal with this? This is hideous looking. I assume it was auto-genned?